pulumi/infra/policies.py

```diff
@@ -8,60 +8,6 @@
 # TODO: Find a way to lazily instantiate these policies; we only need
 # to create them if we need them.
 
-SSM_POLICY: Final[aws.iam.Policy] = aws.iam.Policy(
```

Comment:
Were we not using this?

(🤔 maybe not... I guess we don't have any EC2 instances in Grapl do we?)

Reply:
Nothing in `policies.py` actually lol
but i didn't want to remove the entire thing in this PR since that is, technically, unrelated